lib/mayaHydra/hydraExtensions/CMakeLists.txt

```diff
@@ -146,6 +147,10 @@ set(SRCFILE ${CMAKE_CURRENT_SOURCE_DIR}/mayaHydra.h.src)
 set(DSTFILE ${CMAKE_BINARY_DIR}/include/mayaHydraLib/mayaHydra.h)
 configure_file(${SRCFILE} ${DSTFILE})
 
+set(SRCFILE ${CMAKE_CURRENT_SOURCE_DIR}/mhBuildInfo.h.src)
```

Comment:
Configure the build information file so that it gets build and version information.